meeting-notes/2018/2018-12-03--ipfs-weekly.md

```diff
@@ -0,0 +1,44 @@
+# IPFS All Hands Call December 3, 2018
+
+-   **Moderator:** pkafei
+-   **Notetaker:** olizilla
+-   **Attendees:**
+
+    -   _@jacobheun_
+    -   _@jonnycrunch_
+    -   _@momack2_
+    -   _@terichadbourne_
+    -   _@Mr0grog_
+    -   _@jaycarpenter_
+    -   _@alanshaw_
+
+  
+**Moderator checklist**
+
+-   Ensure that there is a notetaker
+-   Start recording
+-   Call for additional agenda items
+
+  
+**Agenda**
+
+_General discussions, decisions, etc._
+
+-   Announcing event in Berlin tomorrow: Contribute to Open Source Project IPFS - Tuesday, December 4, 2018 6:30 PM to 9:00 PM - https&#x3A;//www.meetup.com/IPFS-Berlin/events/255970865
```

Comment:
@pkafei something in this url is borked :)

Reply:
Where can people follow the event? Is there instructions that the Core Devs could review?

Reply:
It would be great to get the presentation recorded, even if with an phone just so that the core devs can watch and learn from it.

Reply:
[This event won't be live](https://github.com/protocol/event-management/issues/132), but I can record part of it on my phone. I think it would be useful to ask what motivated the attendees to come to the event, and have they ever contributed to open source before. Towards the end of the meetup, I will ask what were some of the pain points of contributing to IPFS and how can we improve the experience.

The event will give us feedback on how we can make our various [open source projects easier to contribute to](https://github.com/protocol/community/blob/22b2755bb647b33fd1432acf959acff93ec9b74c/ROADMAP.md).

Reply:
> This event won't be live, but I can record part of it on my phone. I think it would be useful to ask what motivated the attendees to come to the event, and have they ever contributed to open source before. 

Yes, exactly what I was looking for.

> The event will give us feedback on how we can make our various open source projects easier to contribute to.

Looking forward to learn from all of this feedback. Thank you for putting this event together!